[Bug] [connector-starrocks] CatalogTableUtil.buildWithConfig error

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

when run connector-starrocks e2e test or run pipline  the error will occur

### SeaTunnel Version

2.3.1

### SeaTunnel Config

```conf
source {
  # This is a example source plugin **only for test and demonstrate the feature source plugin**
  StarRocks {
    nodeUrls = ["...:8030"]
    username = root
    password = ""
    database = "ssb"
    table = "lineorder"
    max_retries = 3
    parallelism = 5
    scan_filter = "lo_orderpriority='5-LOW'"
    schema  {
     fields {
       lo_orderkey = "INT"
       lo_linenumber = "INT"
       lo_custkey = "INT"
       lo_partkey = "INT"
       lo_suppkey = "INT"
       lo_orderdate = "INT"
       lo_orderpriority = "STRING"
       lo_shippriority = "INT"
       lo_quantity = "INT"
       lo_extendedprice = "INT"
       lo_ordtotalprice = "INT"
       lo_discount = "INT"
       lo_revenue = "INT"
       lo_supplycost = "INT"
       lo_tax = "INT"
       lo_commitdate = "INT"
       lo_shipmode = "STRING"
        }
    }


}
```


### Running Command

```shell
./bin/start-seatunnel-flink-13-connector-v2.sh --config config/starrocks_thrift_source_connector.test.config -m yarn-cluster -ynm test -yqu dp
```


### Error Exception

```log
The program finished with the following exception:

org.apache.flink.client.program.ProgramInvocationException: The main method caused an error: Schema config need option [schema], please correct your config first
	at org.apache.flink.client.program.PackagedProgram.callMainMethod(PackagedProgram.java:372)
	at org.apache.flink.client.program.PackagedProgram.invokeInteractiveModeForExecution(PackagedProgram.java:222)
	at org.apache.flink.client.ClientUtils.executeProgram(ClientUtils.java:114)
	at org.apache.flink.client.cli.CliFrontend.executeProgram(CliFrontend.java:812)
	at org.apache.flink.client.cli.CliFrontend.run(CliFrontend.java:246)
	at org.apache.flink.client.cli.CliFrontend.parseAndRun(CliFrontend.java:1054)
	at org.apache.flink.client.cli.CliFrontend.lambda$main$10(CliFrontend.java:1132)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1924)
	at org.apache.flink.runtime.security.contexts.HadoopSecurityContext.runSecured(HadoopSecurityContext.java:41)
	at org.apache.flink.client.cli.CliFrontend.main(CliFrontend.java:1132)
Caused by: java.lang.RuntimeException: Schema config need option [schema], please correct your config first
	at org.apache.seatunnel.api.table.catalog.CatalogTableUtil.buildWithConfig(CatalogTableUtil.java:176)
	at org.apache.seatunnel.connectors.seatunnel.starrocks.source.StarRocksSource.prepare(StarRocksSource.java:82)
	at org.apache.seatunnel.core.starter.flink.execution.SourceExecuteProcessor.initializePlugins(SourceExecuteProcessor.java:132)
	at org.apache.seatunnel.core.starter.flink.execution.FlinkAbstractPluginExecuteProcessor.<init>(FlinkAbstractPluginExecuteProcessor.java:67)
	at org.apache.seatunnel.core.starter.flink.execution.SourceExecuteProcessor.<init>(SourceExecuteProcessor.java:58)
	at org.apache.seatunnel.core.starter.flink.execution.FlinkExecution.<init>(FlinkExecution.java:85)
	at org.apache.seatunnel.core.starter.flink.command.FlinkTaskExecuteCommand.execute(FlinkTaskExecuteCommand.java:59)
	at org.apache.seatunnel.core.starter.SeaTunnel.run(SeaTunnel.java:40)
	at org.apache.seatunnel.core.starter.flink.SeaTunnelFlink.main(SeaTunnelFlink.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.flink.client.program.PackagedProgram.callMainMethod(PackagedProgram.java:355)
	... 11 more
```


### Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/source/StarRocksSource.java
+++ b/seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/source/StarRocksSource.java
@@ -78,8 +78,7 @@ public class StarRocksSource
                             getPluginName(), PluginType.SOURCE, mergedConfigCheck.getMsg()));
         }
 
-        Config schemaConfig = pluginConfig.getConfig(CatalogTableUtil.SCHEMA.key());
-        this.typeInfo = CatalogTableUtil.buildWithConfig(schemaConfig).getSeaTunnelRowType();
+        this.typeInfo = CatalogTableUtil.buildWithConfig(pluginConfig).getSeaTunnelRowType();
         this.sourceConfig = SourceConfig.loadConfig(pluginConfig);
     }
 

